extensions/wikia/PageShare/PageShareHooks.class.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class PageShareHooks {
+
+	/**
+	 * Add global JS variable wgAcceptLangList
+	 *
+	 * @param array $vars global variables list
+	 * @return boolean return true
+	 */
+	public static function onMakeGlobalVariablesScript(Array &$vars) {
+		global $wgRequest;
+
+		$wgAcceptLang = $wgRequest->getAcceptLang();
+		$vars['wgAcceptLangList'] = is_array( $wgAcceptLang ) ? array_keys( $wgAcceptLang ) : null;
```

Comment:
I don't see any cache variationing for wikia pages based on the Accept-Language header both in wikia-vcl repo and after checking on sandbox. I'll revert that. Thanks for spotting @bognix, well done! :+1: